<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>698</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Telemetry</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>561</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GRBL Status</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLCDNumber" name="position_y">
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="value" stdset="0">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="grbl_status">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLCDNumber" name="position_x">
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="value" stdset="0">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="status">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Motro controller</string>
      </property>
      <widget class="QPlainTextEdit" name="ramps_output">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>561</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="ramps_input">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>561</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="distance_point1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>200</y>
         <width>51</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">distance_radio_group</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="distance_1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>200</y>
         <width>51</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">distance_radio_group</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="distance_10">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>51</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">distance_radio_group</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="distance_100">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>200</y>
         <width>51</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>100</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">distance_radio_group</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Directional commands</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>351</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QPushButton" name="right_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/right_arrow.svg</normaloff>:/Images/Images/right_arrow.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="left_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/left_arrow.svg</normaloff>:/Images/Images/left_arrow.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="up_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/up_arrow.svg</normaloff>:/Images/Images/up_arrow.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="down_button">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/down_arrow.svg</normaloff>:/Images/Images/down_arrow.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="home_x_button">
          <property name="text">
           <string>X</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/home.svg</normaloff>:/Images/Images/home.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="home_y_button">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/home.svg</normaloff>:/Images/Images/home.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="dwell_button">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Dwell</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="distance_radio_group"/>
 </buttongroups>
</ui>
